source/reference/mongosync.txt

```diff
@@ -38,8 +38,6 @@ Compatibility
 -------------
 
 - .. include:: /includes/fact-no-8.0-support.rst
-
-- .. include:: /includes/fact-minimum-server-version-support.rst
      
   You can migrate data on clusters (source) with versions of MongoDB
```

Comment:
this is the blurb in my previous comment, would like to confirm that this is going away

Reply:
Ah thanks for noting this. I didn't get that the first time around. Removing.